lib/run-app.js

```diff
@@ -0,0 +1,5 @@
+const runApp = (script) => {
+    return script.chunks[4].buf ? script.chunks[4].buf.toString('utf8') : 'run';
+};
+
+module.exports = runApp;
```

Comment:
@jdh7190 Can you move this logic into `is-run-out`? That function can return the name of the run app instead of a boolean

Reply:
Addressed in below commit.